h/templates/client/viewer.html

```diff
@@ -2,6 +2,14 @@
 
 (See gh2642 for rationale for 'ng-show="true"')
  -->
+<selection-tabs ng-if="!search.query() && selectedAnnotationCount() <= 0"
+  tab-selection="tabSelection"
+  total-annotations="totalAnnotations"
+  total-notes="totalNotes"
+  tab-annotations="tabAnnotations"
```

Comment:
As noted above, I think we should remove `tabAnnotations` and `tabNotes` here and just put the constants in a file that can be imported directly from the `selection-tabs` component via a simple import.